c++11/cmake/OpenTracingConfig.cmake

```diff
@@ -0,0 +1,12 @@
+# - Locate the opentracing headers and libraries
+# Usage:
+#    find_package(OpenTracing)
+#    find_package(OpenTracing 0.1.0)  # To specify a minimum version
+# This will create the targets:
+#    OpenTracing::opentracing # dynamic library
+#    OpenTracing::opentracing_static # static library
```

Comment:
The documentation name `OpenTracing::opentracing_static` doesn't match the target name `OpenTracing::opentracing-static`.